client/components/profilePage/allFriendsModal.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState, useEffect } from 'react';
+import { Modal, Image, Button, Table, Badge, Spinner, Row, Col } from 'react-bootstrap';
+import axios from 'axios';
+
+export default function AllFriends({
+  friendsList,
+  handleClose,
+  show,
+}) {
+
+
+
+  return friendsList === undefined ? (
+    <Spinner animation='border' role='status'>
+      <span className='sr-only'>Loading...</span>
+    </Spinner>
+  ) : (
+    <Modal size='lg' show={show} onHide={handleClose} backdrop='static' keyboard={false}>
```

Comment:
Do we want static backdrop?

Reply:
Removed backdrop property on latest push